README.md

```diff
@@ -17,6 +17,7 @@ blog to the most popular websites in the world. PHP is distributed under the
 [![Build status](https://travis-ci.com/php/php-src.svg?branch=master)](https://travis-ci.com/github/php/php-src)
 [![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy/branch/master?svg=true)](https://ci.appveyor.com/project/php/php-src)
 [![Build Status](https://dev.azure.com/phpazuredevops/php/_apis/build/status/php.php-src?branchName=master)](https://dev.azure.com/phpazuredevops/php/_build/latest?definitionId=1&branchName=master)
+[![Push](https://github.com/php/php-src/actions/workflows/push.yml/badge.svg)](https://github.com/php/php-src/actions/workflows/push.yml)
```

Comment:
```suggestion
[![Build status for PHP 8.0](https://github.com/php/php-src/actions/workflows/push.yml/badge.svg?branch=PHP-8.0)](https://github.com/php/php-src/actions/workflows/push.yml?query=branch%3APHP-8.0)
[![Build status for PHP 8.1](https://github.com/php/php-src/actions/workflows/push.yml/badge.svg?branch=PHP-8.1)](https://github.com/php/php-src/actions/workflows/push.yml?query=branch%3APHP-8.1)
[![Build status for PHP 8.2](https://github.com/php/php-src/actions/workflows/push.yml/badge.svg?branch=master)](https://github.com/php/php-src/actions/workflows/push.yml?query=branch%3Amaster)
[![Build status](https://travis-ci.com/php/php-src.svg?branch=master)](https://travis-ci.com/github/php/php-src)
[![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy/branch/master?svg=true)](https://ci.appveyor.com/project/php/php-src)
[![Build Status](https://dev.azure.com/phpazuredevops/php/_apis/build/status/php.php-src?branchName=master)](https://dev.azure.com/phpazuredevops/php/_build/latest?definitionId=1&branchName=master)
```

here is how it is rendered: https://github.com/php/php-src/blob/ff9cf9e4bcb855b67dc21bf27ead57199ba81f51/README.md, the badge annotation is not displayed... maybe we should introduce 3 separate lines for each supported PHP version

GH should be first, it will shortly test Windows as well

and for each line/version, nightly should be 2nd badge and the workflow should be changed, so it is triggered from cron on master on every branch - https://github.com/php/php-src/actions/workflows/nightly.yml?query=branch%3APHP-8.1